### DISCUSSION TITLE
expands fields are not added into include_descendants_tree

### DISCUSSION THREAD

**Discussion by winston0410** [medusajs/medusa#4480] [Q&A] [answered]
I am trying to expand and add `products` into the return query of `/store/product-categories`. I make a query like this:

```ts
fetch(`/store/product-categories?${new URLSearchParams({
  include_descendants_tree: "true",
  expand: "products"
}).toString()}`
```

I notice that the `products` field is added to the categories in the top level, but it is missing in children categories in the descendant tree. Is this the intended behavior?

**Comment by olivermrbl** [comment, marked as answer]
We don't support this due to reasons described [here](https://github.com/medusajs/medusa/issues/3788#issuecomment-1508610090).

To retrieve Products in Categories, you should use the Products API:
```
/store/products?collection_id[]=some_id
```

**Reply by winston0410** [reply]
That's good to know, thanks!
